src/v8runner/src/v8runner.os

```diff
@@ -74,6 +74,14 @@
 	ВыполнитьКоманду(Параметры);
 КонецПроцедуры
 
+Процедура ВыгрузитьКонфигурациюВФайлы(Знач КаталогВыгрузки, Знач ФорматВыгрузки = "Hierarchical") Экспорт
```

Comment:
В смысле 1С в параметрах по умолчанию можно передавать только примитивные
типы, а перечисления/структуры передавать нельзя.
В 1скрипте подозреваю такое же поведение. Или я не прав?
В итоге так легко не удастся заменить параметр по умолчанию.

пт, 13 Май 2016, 0:41 Andrei Ovsiankin <notifications@github.com>:

> In src/v8runner/src/v8runner.os
> <https://github.com/EvilBeaver/oscript-library/pull/93#discussion_r63102426>
> :
>
> > @@ -74,6 +74,14 @@
> >  	ВыполнитьКоманду(Параметры);
> >  КонецПроцедуры
> >
> > +Процедура ВыгрузитьКонфигурациюВФайлы(Знач КаталогВыгрузки, Знач ФорматВыгрузки = "Hierarchical") Экспорт
>
> !resolve
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/EvilBeaver/oscript-library/pull/93/files/63e4739c7011c40f35edec42fd2f116f219dc63c#r63102426>
>